python/cudf/cudf/io/parquet.py

```diff
@@ -669,6 +690,7 @@ def to_parquet(
     index=None,
     partition_cols=None,
     partition_file_name=None,
+    partition_offsets=None,
```

Comment:
We should update the docstring to include the new `partition_offsets` option (unless I missed that change here?)